change-notes/1.25/analysis-python.md

```diff
@@ -1,22 +1,9 @@
 # Improvements to Python analysis
 
-The following changes in version 1.25 affect Python analysis in all applications.
-
-## General improvements
-
-
-## New queries
-
-| **Query**                   | **Tags**  | **Purpose**                                                        |
-|-----------------------------|-----------|--------------------------------------------------------------------|
-
-
-## Changes to existing queries
-
-| **Query**                  | **Expected impact**    | **Change**                                                       |
-|----------------------------|------------------------|------------------------------------------------------------------|
-
-
-## Changes to libraries
-
 * Importing `semmle.python.web.HttpRequest` will no longer import `UntrustedStringKind` transitively. `UntrustedStringKind` is the most commonly used non-abstract subclass of `ExternalStringKind`. If not imported (by one mean or another), taint-tracking queries that concern `ExternalStringKind` will not produce any results. Please ensure such queries contain an explicit import (`import semmle.python.security.strings.Untrusted`).
+* Added model of taint sources for HTTP servers using `http.server`.
+* Added taint modeling of routed parameters in flask.
+* Improved modeling of builtin methods on strings for taint tracking.
+* Improved classification of test files.
+* New class `BoundMethodValue` exposing information about a bound method.
```

Comment:
Sounds a little incomplete to me. How about 

* New class `BoundMethodValue` represents methods bound to an object.

Reply:
Alright, how about this?
```suggestion
* New class `BoundMethodValue` represents a bound method during runtime.
```